src/main/target/NUCLEOF446RE/target.h

```diff
@@ -64,7 +64,7 @@
 #define USE_BARO
 #define USE_FAKE_BARO
 //#define USE_BARO_BMP085
-//#define USE_BARO_BMP280
+#define USE_BARO_BMP280
```

Comment:
Is this an intended change or is this a leftover from testing?

Reply:
This is an intended change; configure at least one commonly used device so that the firmware can be tested without rebuilding.

Reply:
I thought we'd disabled the drivers for physical devices on Nucleo boards in order to make the 'fake' drivers recognised out of the box - have you tested that when leaving this in the fake driver is still used when no physical baro is found?

Reply:
Ah... yeah, okay.